media_mpx.services.yml

```diff
@@ -65,7 +65,14 @@ services:
 
   media_mpx.client_factory:
     class: Drupal\media_mpx\ClientFactory
-    arguments: ['@media_mpx.http_handler_stack', '@http_client_factory']
+    arguments: ['@media_mpx.http_handler_stack', '@http_client_factory', '@cache.media_mpx_http']
+
+  cache.media_mpx_http:
```

Comment:
This splits the responses into a separate cache bin, and core handles creating any database tables automatically. Neat!